keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,44 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
```

Comment:
we will need to write a docstring for this, describing why we need it and how to use it.  This is actually a user facing API.

Reply:
Done